tx/owner.go

```diff
@@ -397,18 +396,7 @@ func (e *WalletOwner) executeTransaction(ctx context.Context, req ExecuteRequest
 	_ = e.updateBalance(ctx)
 
 	if res.Status != StatusOK {
-		p, derr := hexutil.Decode(res.Output)
-		if derr != nil {
-			e.logger.Debug(ctx, "failed to decode the output of the transaction as hex", log.MapFields{
-				"call_type": "DecodeTransactionOutputFailure",
-				"id":        req.ID,
-				"address":   req.Address,
-				"err":       derr.Error(),
-			})
-		}
-
-		output := string(p)
-		msg := fmt.Sprintf("transaction receipt has status %d which indicates a transaction execution failure with error %s", res.Status, output)
+		msg := fmt.Sprintf("transaction receipt has status %d which indicates a transaction execution failure with error %s", res.Status, res.Output)
```

Comment:
Where do you go to interpret the hex string this will print out?

Reply:
the error message is captured by the client, decoded as hex, and decrypted before finally throwing